src/components/App/App.js

```diff
@@ -125,6 +84,14 @@ const App = () => {
             })
     }
 
+    const nominateMovie = movie => {
+        // nominate movie
+        console.log("nominate movie", movie);
```

Comment:
Should we remove the logging statement?

Reply:
Probably should have, but will leave for now.